doc/introduction/circuits.rst

```diff
@@ -114,7 +114,9 @@ devices, the options are:
   and variances analytically. Only possible with simulator devices. Defaults to ``True``.
 
 * ``shots`` (*int*): How many times the circuit should be evaluated (or sampled) to estimate
-  the expectation values. Defaults to 1000 if not specified.
+  the expectation values. Defaults to 1000 if not specified. This argument can be temporarily overwritten
+  when a qnode is called; for example ``myqnode(shots=3)`` will temporarily evaluate ``myqnode``
+  using three shots.
```

Comment:
I'm so excited to get rid of the default of 1000. It was so arbitrary.

Reply:
Whoops it actually defaults to None. Will fix.